lockbox2/programs/liquidity_lockbox/src/lib.rs

```diff
@@ -504,6 +496,7 @@ pub struct DepositPositionForLiquidity<'info> {
   pub position: Box<Account<'info, Position>>,
 
   #[account(mut,
+    address = lockbox.pda_position_account.key(),
```

Comment:
Added a check for the `pda_position_account` to be the one that is in the `lockbox` account.